src/reanimated2/mutables.ts

```diff
@@ -91,8 +75,8 @@ export function makeMutable<T>(
       }
     },
     get value() {
-      if (syncDataHolder) {
-        return NativeReanimatedModule.getDataSynchronously(syncDataHolder);
+      if (NativeReanimatedModule.native) {
+        return executeOnUIRuntimeSync(uiValueGetter)(mutable);
```

Comment:
Can we memoize `executeOnUIRuntimeSync(uiValueGetter)` here so that we don't need to call `NativeReanimatedModule.executeOnUIRuntimeSync` and `makeShareableCloneRecursive` on each read?

Reply:
👍

Reply:
fixed here: https://github.com/software-mansion/react-native-reanimated/pull/5513/commits/04b30b8fa4450770299e5813098ed842f4d9af3b